utils/getMultipleCountries.js

```diff
@@ -0,0 +1,35 @@
+const axios = require('axios');
+const numberFormat = require('./numberFormat');
+const exitCountry = require('./exitCountry');
+const to = require('await-to-js').default;
+const handleError = require('cli-handle-error');
+
+module.exports = async (spinner, table, states, countryName, options) => {
+	if (countryName && !states && !options.chart) {
+		const [err, response] = await to(
+			axios.get(`https://corona.lmao.ninja/v2/countries/${countryName}`)
+		);
+		exitCountry(err, spinner, countryName);
+		err && spinner.stopAndPersist();
+		handleError(`API is down, try again later.`, err, false);
+		const thisCountry = response.data;
+
+		// Format.
+		const format = numberFormat(options.json);
+
+		thisCountry.map(data => table.push([
```

Comment:
This was the output when I've tried to run this:
```
× UNHANDLED ERROR
× ERROR → TypeError
i REASON → thisCountry.map is not a function
i ERROR STACK ↓ 
 TypeError: thisCountry.map is not a function
    at module.exports (C:\Users\lukel\Desktop\development\corona-cli\utils\getMultipleCountries.js:20:15)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at async C:\Users\lukel\Desktop\development\corona-cli\index.js:67:2
```

Reply:
I have fixed this issue in the latest commit, thanks for mentioning. Kindly test and update.